### DISCUSSION TITLE
Reaching out for mentors for project.

### DISCUSSION THREAD

**Discussion by BlossomBreeze25** [openvinotoolkit/openvino#23208] [Google Summer of Code]
Respected sir / ma'am,
I am Ragini Devi Thakkar , 4th Semester BCA(AI) student, want to join your organization. i have knowledge about AI and languages like python , java . html , css . i am interested in this project (4. Add Image-to-Image and Image-to-Text generation with GUI support for OpenVINO GenAI) and I would love to contribute to your organization but could you please tell me how to get started?

Hoping to hear from you soon.

Regards,

Ragini Devi Thakkar.

**Comment by adrianboguszewski** [comment]
@yangsu2022 @FionaZZ92 please look at it

**Comment by yangsu2022** [comment]
Hi,
Thanks for being interested in contributing to OpenVINO C++ GenAI samples for AIPC project.
Besides the related python [OpenVINO notebook](https://github.com/openvinotoolkit/openvino_notebooks/tree/b139f75ef4824064fced7c7784a447f0751c774b/notebooks/235-controlnet-stable-diffusion)  and [GenAI image_generation](https://github.com/openvinotoolkit/openvino.genai/tree/master/image_generation) , OpenVINO [CONTRIBUTING](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md) details and [OpenVINO Docs](https://docs.openvino.ai/2023.3/home.html) will be helpful for learning OpenVINO components.
